### PR TITLE
Build spatial_ref_sys.sql from a data file 

### PR DESCRIPTION
**Pull request by pramsey**
And also build an upgrade file that is included in the extension upgrade script.

### CODE REVIEW

**Comment by strk** [comment]
What's the rationale for the upgrade SQL script being different from the create SQL script ? Wouldn't your proposed upgrade script behave exactly as the create one for a starting empty table ?

File: utils/create_spatial_ref_sys.pl [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+	chop;
+	my ($srid, $auth_name, $auth_srid, $srtext, $proj4text) = split (/\t/);
+	$auth_name =~ s/\'/\'\'/g;
+	$srtext =~ s/\'/\'\'/g;
+	$proj4text =~ s/\'/\'\'/g;
+	print "INSERT INTO \"$SRSTABLE\" (\"srid\", \"auth_name\", \"auth_srid\", \"srtext\", \"proj4text\") VALUES ($srid, '$auth_name', $auth_srid, '$srtext', '$proj4text')";
+	if ($UPDATE) {
+		print "\nON CONFLICT (srid) DO NOTHING";
+	}
+	print ";\n\n";
```
Comment (strk) [inline utils/create_spatial_ref_sys.pl:53]: Do you see any problem in doing this in a single statement, with iteration only adding a new `(...)` record ?
Would `ON CONFLICT` not work correctly in that case ?

**Comment by strk** [comment]
Please make sure builds are still supported out of source tree (Dronie should actually be doing that, if it is running for this PR).

To recap: this change would basically ADD new SRIDs shipped by PostGIS but only if those SRIDs are not already assigned by the user, and would NEVER update existing records (in case, like it happened between some PostGIS versions, they are changed). This change will result in a bigger postgis upgrade script (by how much, roughly?) and a consequent slower upgrade procedure (by how much, roughtly ?)

**Comment by pramsey** [comment]
Superceded by #665

File: utils/create_spatial_ref_sys.pl (hunk 1 above)
Comment (pramsey) [reply utils/create_spatial_ref_sys.pl]: New PR does a multi-valued insert, and the on conflict works, so yay!

**Comment by pramsey** [comment]
> What's the rationale for the upgrade SQL script being different from the create SQL script ? Wouldn't your proposed upgrade script behave exactly as the create one for a starting empty table ?

You're right, an upsert works for both cases, so I've ditched the machinery and done with a minimal patch in #665

**Pull request closed without merging**